src/attributes.jl

```diff
@@ -295,25 +295,33 @@ end
 function get(model::ModelLike, attr::AnyAttribute, args...)
     return get_fallback(model, attr, args...)
 end
+
 function get_fallback(model::ModelLike, attr::AnyAttribute, args...)
     return throw(
         ArgumentError(
-            "ModelLike of type $(typeof(model)) does not support accessing the attribute $attr",
+            "Unable to get `$(attr)` with the provided arguments. Either " *
+            "$(typeof(model)) does not support the attribute, or the " *
+            "provided arguments are incorrect.",
```

Comment:
It would be better to distinguish the cases. We just have to write 3 methods: one for model/solver attributes, one for variables attributes and one for constraint attibutes to make sure we don't catch any incorrect arguments. I agree that the current is confusing and I get it a lot. Having it say it can be 2 different things is less helpful.